root/edit/components/Vote.js

```diff
@@ -42,7 +42,7 @@ const VoteCheckbox = ({
     ? getLatestVoteForEditor(edit, user)
     : null;
   const checked =
-    (latestVote && latestVote.vote == props.value) ||
+    (latestVote && latestVote.vote === props.value) ||
```

Comment:
I was wondering if I had it like that because of Perl serialization issues. But I see we do `$self->vote + 0` in `Entity::Vote`, so it should be fine...